main.c

```diff
@@ -18,7 +19,7 @@ static int init(App *app, Config *cfg)
     return 0;
   }
 
-#ifndef __WIN32
+#ifdef __LINUX
```

Comment:
I've also found out, that `__LINUX` is not defined by default. The standard predefined macro is `__linux__`.

Reply:
Yeah, I detect the platform in the Makefile using `uname` and set the define based on that. So when running make on Linux, __LINUX will be defined. Not sure whether there is a well-defined builtin define for every platform that works with both gcc and clang

Reply:
Ah okay, my bad